packages/web-components/src/components/masthead/masthead.scss

```diff
@@ -469,10 +485,29 @@ $breakpoint--desktop-nav: 960px;
   @extend :host(#{$prefix}-side-nav-menu);
 }
 
-:host(#{$dds-prefix}-left-nav-menu-item) {
+:host(#{$dds-prefix}-left-nav-menu-item),
+:host(#{$dds-prefix}-left-nav-cta-item) {
   @extend :host(#{$prefix}-side-nav-menu-item);
 }
 
+:host(#{$dds-prefix}-left-nav-cta-item) {
+  @include carbon--breakpoint(md) {
+    display: none;
+  }
+
+  a.#{$prefix}--side-nav__link .#{$prefix}--side-nav__link-text {
+    @include carbon--type-style(body-short-01, true);
```

Comment:
@proeung @oliviaflory the font difference is a bit odd. I wasn't too involved in the specs for L0.0 but wondering if it was kept the same font size as desktop and just was never updated in mobile. @oliviaflory do you know if this should remain `$body-short-01` or should we switch to `$body-short-02` ?

Reply:
@proeung Just reviewed this with @oliviaflory and this is incorrect. It should match all of the type above it and be `$body-short-02,` however in the L0 v2.0.0 handoff today, we're going to update all of the styles on mobile to `$body-short-01`. It's it relatively an easy switch to make?

Reply:
> @proeung Just reviewed this with @oliviaflory and this is incorrect. It should match all of the type above it and be `$body-short-02,` however in the L0 v2.1.0 handoff today, we're going to update all of the styles on mobile to `$body-short-01`. It's it relatively an easy switch to make?

@jwitkowski79 (cc @oliviaflory @proeung) The type styles for the mobile nav are in need of some housekeeping. Since that's going to take some time, I'll simply update the mobile CTAs to use `body-short-02` for now to keep things consistent and address the overall type style changes in the v2.1.0 round of work. Sound good?

Reply:
@jkaeser Sounds good!